tests/cpp/test_host_ir_jit.cpp

```diff
@@ -326,59 +326,14 @@ TEST_F(HostIrJitTest, Matmul) {
 
   TensorView* tv0 = makeContigTensor(3);
   TensorView* tv1 = makeContigTensor(3);
-  TensorView* tv2 = makeContigTensor(3);
-  auto* matmul = IrBuilder::create<MatmulOp>(tv2, tv0, tv1);
-
-  hic->addInput(tv0);
-  hic->addInput(tv1);
-  hic->addInput(tv2);
-  hic->addOutput(tv2);
```

Comment:
While it's valid host IR, we don't emit MatmulOp this way in reality. The new test is more realistic. In the future, we could replace this test with a FusionExecutorCache-level test with HostIrLowering and HostIrJit both enabled.

Reply:
What is missing before we switch this test to FusionExecutorCache with the knobs turned on?